science/rtl_433/Portfile

```diff
@@ -4,10 +4,7 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 PortGroup           github 1.0
 
-github.setup        merbanan rtl_433 18.05
-version             18.05
-epoch               1
```

Comment:
Also, the `epoch` line cannot be deleted or else no one who has it installed will see any updates going forward.

Reply:
12 > 05 or I misunderstood the meaning of `epoch`

from macports guide
```
An optional keyword (default value is 0) that must be used when a port is updated to a version that is numerically less than the previous version, for example 1.10 -> 1.2
```

Reply:
Once set, it can never be removed.

Reply:
Or reduced really.

Reply:
The precedence for checking for updates is epoch -> version -> revision. That allows one to downgrade to a previous version. But the side effect is that the epoch must stick around once it is changed from 0 - the default. Yes, the docs are not clear on that.

Reply:
done. this statement should be included in the guide. I will send a PR with your phrase.